src/CodePen.js

```diff
@@ -12,5 +12,5 @@ export const shouldTransform = url => {
 export const getHTML = url => {
   const iframeUrl = url.replace('/pen/', '/embed/preview/');
 
-  return `<iframe src="${iframeUrl}" style="width:100%; height:300px;"></iframe>`;
+  return `<iframe src="${iframeUrl}" style="width: 100%;" height="400" scrolling="no" frameborder="no" allowtransparency="true" allowfullscreen="true"></iframe>`;
```

Comment:
```suggestion
  return `<iframe src="${iframeUrl}" style="width:100%; height:300px;"></iframe>`;
```

Don't forget to update the test snapshot too 😉